### DISCUSSION TITLE
Execute a script at boot

### DISCUSSION THREAD

**Discussion by RichardBachelier** [ltsp/ltsp#897] [General]
Hello,
I have a file named test.tar.gz in the directory /etc/ltsp/initrd/bin/.
Afterward, I run the 'ltsp initrd' command to apply the changes.
During the system startup, I would like to execute the command 'tar xfzp /etc/ltsp/initrd/bin/test.tar.gz -C /etc/ltsp/initrd/bin/'.
I added the instruction POSTE_APPLET_TAR="tar xfzp /etc/ltsp/initrd/bin/test.tar.gz -C /etc/ltsp/initrd/bin/" in the ltsp.conf file.

It did not work, and I believe I may not have understood the subtlety of this instruction.
Questions:

How does this command work?
Is this the best method in this case, or should I use another mechanism?
Thank you for your assistance.

Richard

**Comment by alkisg** [comment]
The directive you're looking for is called POST_INIT_TAR="...".

And an even easier way to do it is to just put the things you want in /etc/ltsp/initrd/bin/ without compressing them, without using a tar. Then you don't need any commands at all, they'll be there after you run `ltsp initrd` and reboot your clients.
